### PR TITLE
Fixed deleting user

### PR DESCRIPTION
**Pull request by Cheshiriks**
Fixed deleting user

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt [hunk 1]
```diff
@@ -23,6 +23,27 @@ import kotlinx.browser.window
...
+        )
+        if (response.ok) {
+            val replyToLogout = post(
+                "${window.location.origin}/logout",
+                Headers(),
+                "ping",
+                loadingHandler = ::loadingHandler,
+            )
+            if (replyToLogout.ok) {
+                window.location.href = "${window.location.origin}/#"
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt:41]: Are you sure about url?

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt]: Fixed it

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt [hunk 2]
```diff
@@ -23,6 +23,27 @@ import kotlinx.browser.window
 val deleteSettingsCard: FC<SettingsProps> = FC { props ->
     val deleteUserWindowOpenness = useWindowOpenness()
 
+    val deleteUser = useDeferredRequest {
+        val response = get(
+            url = "$apiUrl/users/delete/${props.userInfo?.name!!}",
+            headers = jsonHeaders,
+            loadingHandler = ::loadingHandler,
+            responseHandler = ::noopResponseHandler,
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt:31]: To my mind, a response handler is needed here

File: save-frontend/webpack.config.d/dev-server.js [hunk 3]
```diff
@@ -1,54 +1,14 @@
 config.devServer = Object.assign(
```
Comment (sanyavertolet) [inline save-frontend/webpack.config.d/dev-server.js:1]: Do we now want to make everyone start a gateway to test locally?

Comment (Cheshiriks) [reply save-frontend/webpack.config.d/dev-server.js]: removed it. I merged it by accident.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt]: Is there a response handler here?

**Pull request merged**